### ISSUE TITLE
Tampermonkey breaks iframe with `frame-ancestors` and `x-frame-options deny`

### ISSUE THREAD

**Issue by mmis1000** [Tampermonkey/tampermonkey#1447]
1. Make a script which logs a line that target all amd website pages

2. open this link https://www.amd.com/en/technologies/radeon-software

## Expected Behavior

Nothing happen except a console message

## Actual Behavior

All media content did not get loaded

## Specifications

- Firefox 100
- TM: 4.15.6154
- OS: Windows 11

## Script

(Please give an example of the script if applicable.)

```js
// ==UserScript==
// @name         New Userscript
// @namespace    http://tampermonkey.net/
// @version      0.1
// @description  try to take over the world!
// @author       You
// @match        https://www.amd.com/*
// @icon         https://www.google.com/s2/favicons?sz=64&domain=amd.com
// @grant        none
// ==/UserScript==

(function() {
    'use strict';
    console.log('test')
    // Your code here...
})();
```

See also: https://github.com/webcompat/web-bugs/issues/100949#issuecomment-1069121459

**Comment by KTibow** [comment]
I'm still getting this issue with the beta.

**Comment by KTibow** [comment]
Nevermind, I needed to restart Firefox first.

**Comment by KTibow** [comment]
Now it doesn't work again :shrug:

**Comment by KTibow** [comment]
For context, this is breaking [r/place](https://www.reddit.com/r/place/).
```js
// ==UserScript==
// @name         for r/place
// @namespace    http://tampermonkey.net/
// @version      0.1
// @description  try to take over the canvas!
// @author       KTibow
// @match        https://hot-potato.reddit.com/*
// @icon         https://www.google.com/s2/favicons?sz=64&domain=reddit.com
// @grant        none
// @run-at       document-end
// ==/UserScript==
alert("hi");
```

**Comment by derjanb** [comment]
@KTibow Yes, the fix was incomplete. Sorry for any inconvenience.

**Comment by derjanb** [comment]
Should be fixed at 4.16.6160 ([xpi](https://firefox.tampermonkey.net/firefox-current-beta.xpi))

Please install the [BETA version](https://www.tampermonkey.net/index.php?browser=firefox) or check for BETA version updates at `about:addons` 

For a quick fix please export your settings and scripts as zip or (JSON) file at the "Utilities" tab and import it back at the fixed BETA version.
